src/web/components/SubNav/SubNav.tsx

```diff
@@ -98,7 +98,7 @@ const linkStyle = css`
 `;
 
 const selected = css`
-	font-weight: 700;
+	font-weight: 700 !important;
```

Comment:
`!important` is almost always a risky solution, I would try and ensure that the cascade is followed instead, so the order in which variables are added I think means they'll be added in the CSS